src/lib.rs

```diff
@@ -11,19 +11,95 @@
 //! On the write side, `async-bincode` buffers the serialized values, and asynchronously sends the
 //! resulting bytestream.
 #![deny(missing_docs)]
```

Comment:
We should document here that all the relevant types are in the sub-modules, and direct people towards the appropriate one for their use-case.

Reply:
Will do.